<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>533</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>838</width>
    <height>533</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>838</width>
    <height>533</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VDraw</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>591</width>
      <height>491</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>585.000000000000000</width>
      <height>400.000000000000000</height>
     </rectf>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>231</width>
      <height>123</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="lineButton">
       <property name="text">
        <string>Line</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="circleButton">
       <property name="text">
        <string>Circle</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="ellipseButton">
       <property name="text">
        <string>Ellipse</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="rectButton">
       <property name="text">
        <string>Rectangle</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>229</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="1" column="1">
      <widget class="QPushButton" name="lineColorButton">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Line color</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="zoomInButton">
       <property name="text">
        <string>Zoom +</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="zoomOutButton">
       <property name="text">
        <string>Zoom -</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Line width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="penWidthSB">
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Line type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="lineTypeList">
         <item>
          <property name="text">
           <string>Solid</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dash</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dot</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dash dot</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dash dot dot</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Clear field</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="fillColorButton">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Fill color</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>graphicsView</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionBegin"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionBegin">
   <property name="text">
    <string>Begin</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
